public_html/wp-content/plugins/wc-post-types/.eslintrc.js

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
If we use core's eslint rules instead of our own customizations, I think we can avoid adding these .eslintrc files to every sub-project… or maybe we can add it to the top-level of the project? I can look into this more.